tests/baselines/reference/abstractPropertyNegative.errors.txt

```diff
@@ -31,10 +31,12 @@ tests/cases/compiler/abstractPropertyNegative.ts(41,18): error TS2676: Accessors
         abstract m(): string;
         abstract get mismatch(): string;
                      ~~~~~~~~
-!!! error TS2380: 'get' and 'set' accessor must have the same type.
+!!! error TS2380: 'get' and 'set' accessor must have the same type 'number', but this 'get' accessor has the type 'string'.

```

Comment:
I think the elaboration `but this 'get' accessor has the type 'string' is not needed. adding a related info is sufficient.